formats/world_location_news.jsonnet

```diff
@@ -28,4 +28,8 @@
       },
     },
   },
+  links: (import "shared/base_links.jsonnet") + {
+    ordered_contacts: "Contact details for this World Location",
```

Comment:
Could we signpost in the description here that this is only used for international delegations? I'm thinking forward to a future in which world location news != international delegation

Reply:
Good idea.  I've added this to the comment.